src/components/Sum.jsx

```diff
@@ -1 +1,9 @@
-// export function Sum() {}
+import React from 'react';
+
+export default function Sum({ a = 0, b = 0 }) {
+  return (
+    <p>
+    {`Sum of ${a} and ${b} is ${a + b}`}
+  </p>
```

Comment:
Pay attention on indents
```suggestion
    <p>
      {`Sum of ${a} and ${b} is ${a + b}`}
    </p>
```